File: game-server/src/main/java/work/fking/pangya/game/packet/outbound/LobbyResponses.java
package work.fking.pangya.game.packet.outbound;

import work.fking.pangya.networking.protocol.OutboundPacket;

public class LobbyResponses {

    public static OutboundPacket ackLeave() {
        return buffer -> buffer.writeShortLE(0xf6);
    }
}

File: game-server/src/main/kotlin/work/fking/pangya/game/packet/outbound/LobbyResponses.kt
package work.fking.pangya.game.packet.outbound

import work.fking.pangya.networking.protocol.OutboundPacket

object LobbyResponses {
    fun ackLeave(): OutboundPacket {
        return OutboundPacket { buffer -> buffer.writeShortLE(0xf6) }
    }
}
